### ISSUE TITLE
Entity hit knockback

### ISSUE THREAD

**Issue by iTrooz** [snackbag/TT20#9]
Hey, I was going through the list of features, and I think a big one is missing: the knockback entities take when hit (which is slower when ticks are low)
Personally, this is the biggest symptom of lag I experience

**Comment by VicoIlFico** [comment]
> Hey, I was going through the list of features, and I think a big one is missing: the knockback entities take when hit (which is slower when ticks are low) Personally, this is the biggest symptom of lag I experience

I strongly agree. Seeing entities lag, especially when you hit them and have to wait 3-5 seconds for them to move and being able to hit them again is by far the most annoying symptom of tps lag. I don't know if doing anything about it is possbile but if it is it would definitely be one of the best features this mod offers

**Comment by LuckserSky34** [comment]
https://github.com/txmydev/NachoSpigot/blob/master/NachoSpigot-Server/src/main/java/dev/cobblesword/nachospigot/hitdetection/LagCompensator.java

its a patch form 1.8.8 but i think can work with newer versions. this is implemented in some plugins that modify knockbacks and hit detection in pvp servers

**Comment by JXSnack** [comment]
Moved to #20
